examples/README.md

```diff
@@ -0,0 +1,70 @@
+# Datashader Examples
+
+The examples rely on some sample data. To download the data, run the
+`download_sample_data.py` script. This may take up to 20 minutes, even on a
+good network connection. The dataset is roughly 1.5 GB on disk.
+
+```
+python download_sample_data.py
+```
+
+The examples also require bokeh to be installed. Bokeh is available through
+either conda or pip.
+
+```
+conda install bokeh
+
+or
+
+pip install bokeh
+```
+
+## Examples
+
+### Dashboard
+
+An example interactive dashboard using [bokeh
+server](http://bokeh.pydata.org/en/latest/docs/user_guide/server.html)
+integrated with a datashading pipeline. To start, run:
+
+```
+python dashboard/dashboard.py --config dashboard/nyc_taxi.yml
+```
+
+### Notebooks
+
+A few example notebooks are also included. To run, install and startup a
```

Comment:
The filled-out notebooks are 42MB at present, which is more than I want to put into this repo, but of course they could go into some other repo.